presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -284,6 +287,27 @@ This table can then be queried in Presto::
 
     SELECT * FROM hive.web.page_view;
 
+.. _clustered-tables:
+
+Clustered hive tables support
+-----------------------------
+
+By default presto supports only one data file per bucket per partition for clustered tables (Hive tables declared with ``CLUSTERED BY`` clause).
+If number of files does not match number of buckets exception would be thrown.
+
+The enable support for cases when multiple INSERTs where done to single partition of clustered user may use:
```

Comment:
Maybe rewrite this line to "To enable support for cases where there are more than one file per bucket, when multiple INSERTs are done to a single partition of the clustered table, you can use:"